test/tests/bootstrap/test_redfish10_os_bootstrap_base.py

```diff
@@ -19,11 +55,8 @@
 NODECATALOG = fit_common.node_select()
 
 NODE = ""
-# Select one node at random that's not a management server
-for dummy in NODECATALOG:
-    NODE = NODECATALOG[random.randint(0, len(NODECATALOG)-1)]
-    if fit_common.rackhdapi('/api/2.0/nodes/' + NODE)['json']['name'] != "Management Server":
-        break
+# Select one node at random
+NODE = NODECATALOG[random.randint(0, len(NODECATALOG)-1)]
```

Comment:
Yes, the management server now has a different node type which gets filtered out by node_select()